lib/server.js

```diff
@@ -17,7 +17,10 @@ function Server(options) {
     EventEmitter.call(this, options);
 
     this.server = dgram.createSocket('udp4');
-    this.server.on('message', parser.parse.bind(this));
```

Comment:
Also here just put a space between `){` to keep the style consistent and I'll merge it in asap. Thanks!